src/TwigYard/Component/MailerMessageBuilder.php

```diff
@@ -14,7 +14,7 @@ class MailerMessageBuilder
      */
     public function __construct()
     {
-        $this->message = Swift_Message::newInstance();
+        $this->message = new Swift_Message();
```

Comment:
Could we pass this in by DI?

Reply:
At afternoon meeting, we decided that we will not pass pure components (like Mailer, ConfigCache etc) to the DI container.